state/factory.go

```diff
@@ -54,11 +54,11 @@ type (
 	Factory interface {
 		lifecycle.StartStopper
 		// Accounts
-		LoadOrCreateState(string, uint64) (*State, error)
```

Comment:
LoadOrCreateState() and CachedState() will be removed from factory interface
so you can leave them as is, to save merge/conflict hassle later